src/vscode-bicep/src/extension.ts

```diff
@@ -12,7 +12,7 @@ import {
   BuildCommand,
   CommandManager,
   DeployCommand,
-  ForceModulesRestoreCommand,
```

Comment:
Need to restore `ForceModulesRestoreCommand`.

Reply:
just put it back, thanks 👍